client/src/constants/algolia_config.js

```diff
@@ -0,0 +1,8 @@
+import algoliasearch from 'algoliasearch/lite';
```

Comment:
why are we using algoliasearch/lite instead of regular Algolia?

Reply:
lite is much faster and we don't need the full functionality for MVP

Reply:
Smart, good idea.